pallets/zk-snarks/src/tests.rs

```diff
@@ -200,19 +203,19 @@ fn test_verification_success() {
 			prepare_correct_public_inputs_json().as_bytes().into(),
 			vk.as_bytes().into()
 		));
-		assert_ok!(ZKSnarks::verify(RuntimeOrigin::none(), proof.as_bytes().into()));
+		assert_ok!(ZKSnarks::verify(RuntimeOrigin::signed(ALICE_ACCOUNT_ID), proof.as_bytes().into()));
```

Comment:
formatter formats this file, please run it so we can merge properly formatted code